image_resizer.py

```diff
@@ -0,0 +1,85 @@
+import os, argparse
+from PIL import Image
+
+pars = argparse.ArgumentParser(description='Image Resizer parameters.')
+# Arguments definition
+
+pars.add_argument('path_in', metavar='PATH_IN', type = str, nargs = 1,
+                  help='Path of folder with input images.')
+pars.add_argument('path_out', metavar='PATH_OUT', type=str, nargs = 1,
+                  help='Path of folder with output images.')
+pars.add_argument('width', metavar='WIDTH', type=int, nargs=1,
+                  help='Width of image (px). NOTE: Height will be \
+                  proportional to Width.')
+pars.add_argument('-wat', metavar=('WATERMARK_PATH'), type=str, nargs= 1,
+                  help='Path of folder with watermark')
+pars.add_argument('--pos', metavar=('WATERMARK_POSITION.'), type=str, nargs=1,
+                  help='Position of Watermark on image: 1: left above. \
+                  2: right above. 3: left below. 4: right below. DEFAULT: 4: right below.')
+args = pars.parse_args()
+
+path_in = args.path_in # Path of folder with input images
+path_out = args.path_out # Path of folder with output images
+width = args.width[0] # Width of image (px). NOTE: Height will be proportional to Width
+folder_in = os.listdir(path_in[0])
+
+# Path of watermark and position of watermark on image
+path_water, pos_water = None, None
+
+# Check if optional arguments are exist
+if args.wat != None:
```

Comment:
I've just read PEP 8 once again and found : 
"Comparisons to singletons like None should always be done with is or is not , never the equality operators."
and why it's important: https://pythonworld.ru/tipy-dannyx-v-python/none.html